[BUG] Getting started documentation seems outdated

I tried to follow the instructions at https://mc-stan.org/stanc3/stanc/getting_started.html on OS X with dune version 2.9.1, opam version 2.1.2, ocaml version 4.13.1 and they did not work.

### Current Behavior:
> opam install --deps-only --with-test .
[WARNING] Failed checks on stanc package definition from source at
          git+file:///Users/matthijs/Desktop/Matthijs/Research/stanc3#master:
    error 23: Missing field 'maintainer'
  warning 25: Missing field 'authors'
  warning 35: Missing field 'homepage'
  warning 36: Missing field 'bug-reports'
[ERROR] Package conflict!
  * Missing dependency:
    - ocamlformat = 0.19
    no matching version



> bash -x scripts/setup_dev_env.sh
+ bash -x ./install_ocaml.sh
bash: ./install_ocaml.sh: No such file or directory
+ bash -x ./install_build_deps.sh
bash: ./install_build_deps.sh: No such file or directory
+ bash -x ./install_dev_deps.sh
bash: ./install_dev_deps.sh: No such file or directory
++ opam env
+ eval 'OPAM_SWITCH_PREFIX='\''/Users/matthijs/.opam/4.13.1'\'';' export 'OPAM_SWITCH_PREFIX;' 'CAML_LD_LIBRARY_PATH='\''/Users/matthijs/.opam/4.13.1/lib/stublibs:/Users/matthijs/.opam/4.13.1/lib/ocaml/stublibs:/Users/matthijs/.opam/4.13.1/lib/ocaml'\'';' export 'CAML_LD_LIBRARY_PATH;' 'OCAML_TOPLEVEL_PATH='\''/Users/matthijs/.opam/4.13.1/lib/toplevel'\'';' export 'OCAML_TOPLEVEL_PATH;' 'PATH='\''/Users/matthijs/.opam/4.13.1/bin:/Users/matthijs/.cabal/bin:/Users/matthijs/.ghcup/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin'\'';' export 'PATH;'
++ OPAM_SWITCH_PREFIX=/Users/matthijs/.opam/4.13.1
++ export OPAM_SWITCH_PREFIX
++ CAML_LD_LIBRARY_PATH=/Users/matthijs/.opam/4.13.1/lib/stublibs:/Users/matthijs/.opam/4.13.1/lib/ocaml/stublibs:/Users/matthijs/.opam/4.13.1/lib/ocaml
++ export CAML_LD_LIBRARY_PATH
++ OCAML_TOPLEVEL_PATH=/Users/matthijs/.opam/4.13.1/lib/toplevel
++ export OCAML_TOPLEVEL_PATH
++ PATH=/Users/matthijs/.opam/4.13.1/bin:/Users/matthijs/.cabal/bin:/Users/matthijs/.ghcup/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin
++ export PATH
++ dune external-lib-deps --missing src/stanc/stanc.exe
++ tail -n1
++ cut -c 12-
+ install_missing='all ppx_jane'
+ echo 'Trying to install missing deps for stanc with all ppx_jane'
Trying to install missing deps for stanc with all ppx_jane
+ all ppx_jane
scripts/setup_dev_env.sh: line 15: all: command not found


### Expected Behavior:
I would expect either of these commands to install all required dependencies without errors.

### Environment:
<!--
Example:
- OS: OSX 10.15.7 (19H1419)
- dune version 2.9.1
- opam version 2.1.2
- ocaml version 4.13.1


### Other remarks:
Updating stanc.opam to  the following 

```
# This file is generated by dune, edit dune-project instead
opam-version: "2.0"
synopsis: "The Stan compiler and utilities"
depends: [
  "dune" {>= "2.8"}
  "ocaml" {= "4.12.0"}
  "core_kernel" {= "v0.14.2"}
  "menhir" {= "20210929"}
  "ppx_deriving" {= "5.2.1"}
  "fmt" {= "0.8.8"}
  "yojson" {= "1.7.0"}
  "ocamlformat" {with-test & = "0.19"}
  "merlin" {with-test & = "4.3.1"}
  "utop" {with-test}
  "ocp-indent" {with-test}
  "patdiff" {with-test}
  "odoc" {with-doc}
]
build: [
  ["dune" "subst"] {dev}
  [
    "dune"
    "build"
    "-p"
    name
    "-j"
    jobs
    "@install"
    "@runtest" {with-test}
    "@doc" {with-doc}
  ]
]
``` 

and 
running
> bash -x setup_dev_env.sh
from the scripts directory fixes the problem for me.

--- a/docs/getting_started.mld
+++ b/docs/getting_started.mld
@@ -14,11 +14,11 @@ installed via apt, homebrew, or your system's package manager.
  {- Download the repository (or your fork of it) from Github}
  {- Install the necessary {!page-dependencies} by running
 
-[bash -x scripts/setup_dev_env.sh].
+[cd scripts; bash -x ./setup_dev_env.sh].
 
    If you already have opam installed, you can instead run
 
-[opam install --deps-only --with-test .]
+[opam update; opam install --deps-only --with-test .]
 
    If you wish to {e install} stanc on your system, you can run the above
 [opam] command without [--deps-only].}
@@ -156,5 +156,3 @@ If you use {{:https://github.com/jwiegley/use-package}[use-package.el]}, this sn
             (define-key tuareg-mode-map (kbd "C-M-<tab>") #'ocamlformat)
             (add-hook 'before-save-hook #'ocamlformat-before-save)))
 ]}
-
-

